### PR TITLE
Reply so gitpull command doesn't error out on user interface

### PR DESCRIPTION
**Pull request by CircuitSacul**

### CODE REVIEW

File: mcoding_bot/cogs/owner.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ async def gitpull(self, ctx: pincer.objects.MessageContext):
         if ctx.author.id not in self.bot.config.owner_ids:
             return "You don't have permission to do this."
         os.system("git pull")
+        await ctx.reply("Pulled from github, restarting bot...")
```
Comment (Sigmanificient) [inline mcoding_bot/cogs/owner.py:26]: Would be nice to have answer after the pull

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: mcoding_bot/cogs/owner.py (hunk 1 above)
Comment (CircuitSacul) [reply mcoding_bot/cogs/owner.py:26]: It is after the pull

Comment (CircuitSacul) [reply mcoding_bot/cogs/owner.py:26]: Though we probably need to defer so it doesn't timeout before the pull

Comment (Sigmanificient) [reply mcoding_bot/cogs/owner.py:26]: to avoid timeout use `await ctx.ack()` at top of the func

Comment (CircuitSacul) [reply mcoding_bot/cogs/owner.py:27]: Done

**Pull request merged**